pkg/option/config.go

```diff
@@ -773,6 +773,9 @@ const (
 	// ServiceLoopbackIPv4 is the address to use for service loopback SNAT
 	ServiceLoopbackIPv4 = "ipv4-service-loopback-address"
 
+	// ServiceLoopbackIPv6 is the address to use for service loopback SNAT
+	ServiceLoopbackIPv6 = "ipv6-service-loopback-address"
```

Comment:
Sorry for the delay on this comment. We prefer that all new config options are added to a hive cell, to begin the process of modularizing the related feature. If you take a look at the [history on this file](https://github.com/cilium/cilium/commits/main/pkg/option/config.go), you will see some recent example of refactors into cells. The initial refactor can be purely for the sake of config, with other modularlization coming later. Could you look into making this change?

Reply:
Logically, pkg/ipam/cell/cell.go seems like the right place, but given the breadth of changes that are required in this PR, that might not be feasible on the dependency graph. Could you give that a try and see if you run into any issues?

@joamaki is there a good way to visualize the dependency graph in order to make this type of decision, or any other advice that you would provide here?

Reply:
Since we already have `ServiceLoopbackIPv4` here I'd be OK with adding it here for now and then moving them both into a saner place in another PR. It'd probably make sense to do that as part of a bigger refactoring that moves `daemon/cmd/ipam.go` into `pkg/ipam`. @cilium/sig-ipam any opinion?